### PR TITLE
buffered discord logger

### PR DESCRIPTION
**Pull request by Fogapod**

### CODE REVIEW

File: usautobuild/logger.py [hunk 1]
```diff
@@ -51,14 +56,14 @@ def setup_logger(arg_level: str) -> None:
 def setup_extra_loggers(config: Config) -> None:
     """Configure complex loggers requiring config"""
 
-    discord_webhook = config.discord_webhook
-    if discord_webhook is not None:
-        handler = DiscordHandler(discord_webhook)
+    if (discord_webhook := config.discord_webhook) is not None:
```
Comment (corp-0) [inline usautobuild/logger.py:59]: Forgot how to read the walrus, what is the unsugared version of this code?

Comment (Fogapod) [reply usautobuild/logger.py:59]: same as before

Comment (corp-0) [reply usautobuild/logger.py:59]: I mean, the walrus is a shorthand for something else. Sugar syntax.

Comment (corp-0) [reply usautobuild/logger.py:59]: Ah I understood what you meant.

**Review by corp-0 — APPROVED** [review]

File: usautobuild/logger.py (hunk 1 above)
Comment (Fogapod) [reply usautobuild/logger.py:59]: i just changed non walrus to walrus

**Pull request merged**